src/View/Helper/AbstractHelper.php

```diff
@@ -558,4 +559,32 @@ protected function hasAllowedPrefix(string $attribute): bool
 
         return false;
     }
+
+    /**
+     * translate the label
+     */
+    protected function translateLabel(string $label): string
+    {
+        if (null !== ($translator = $this->getTranslator())) {
+            $label = $translator->translate(
+                $label,
+                $this->getTranslatorTextDomain()
+            );
+        }
+
+        return $label;
```

Comment:
`return $this->getTranslator()?->translate($label, $this->getTranslatorTextDomain()) ?? $label;` or such? Just wondering if it makes sense, as it potentially adds an always-executed method call :-)

Reply:
Actually, it does not call the method, if no translator is set:

```php
<?php

function doesSomethingAndReturns(): string {
    echo 'called';

    return 'haha';
}

$instance = null;

echo $instance?->thisWillNeverBeCalled(doesSomethingAndReturns()) ?? 'alternative';
```

https://3v4l.org/chNbV